### DISCUSSION TITLE
include_tables not including tables in non dbo object

### DISCUSSION THREAD

**Discussion by nishantpandey107** [sqlalchemy/sqlalchemy#10319] [Usage Questions] [answered]
I am connecting to SQLServer. 
while creating db object from SQLDatabase, i am including some tables name which are not under [dbo] object. but it's not accepting

i tried following
db = SQLDatabase(engine, include_tables=['<tableName>'], sample_rows_in_table_info=5)
db = SQLDatabase(engine, include_tables=['<NON-DBO>']['<tableName>'], sample_rows_in_table_info=5)

none of the combination working. 
if I don't include "include_tables", it's including all the tables in dbo schema, but none from non-dbo schema

**Comment by CaselIT** [comment]
Hi,

I don't know what `SQLDatabase` is. There is nothing in sqlalchemy that's called that.
Are you sure you are asking to the correct library?

**Reply by zzzeek** [reply]
it's likely part of an internal framework that uses SQLAlchemy, that is doing something like automap and/or what SQLSoup used to do.

**Comment by nishantpandey107** [comment, marked as answer]
thanks for reply anyway @zzzeek @CaselIT ... i got my answer

This is the fix
engine = create_engine('your_database_uri')
db = SQLDatabase(engine, schema='your_schema_name')
